github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- name: periodic-kubevirtci-sync-crio-repository-weekly
+  interval: 168h
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/gcs/service-account.json
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - >
+            set -e;
+            BUCKET=kubevirtci-crio-mirror
+            LOCAL_MIRROR_DIR=opensuse-mirror
+            OS=CentOS_8_Stream
+            CRIO_VERSION=1.20
+
+            DEVEL_LIBCONTAINERS_STABLE=devel_kubic_libcontainers_stable
+            curl -L -o $DEVEL_LIBCONTAINERS_STABLE.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+            reposync -c $DEVEL_LIBCONTAINERS_STABLE.repo -p ./$LOCAL_MIRROR_DIR -n --repoid=$DEVEL_LIBCONTAINERS_STABLE
```

Comment:
There's repetition here, could you modify `sync_crio_repo_for_version` to take `DEVEL_LIBCONTAINERS_CRIO` as parameter? then call it twice, first with `devel_kubic_libcontainers_stable` and second with `devel_kubic_libcontainers_stable_cri-o_${CRIO_VERSION}`

Reply:
Updated, although in a slightly different way than you suggested, let me know what you think

Reply:
lgtm :+1: